services/navigation/builtin/explore.go

```diff
@@ -18,9 +18,8 @@ func (svc *builtIn) startExploreMode(ctx context.Context) {
 	svc.logger.CDebug(ctx, "startExploreMode called")
 
 	svc.activeBackgroundWorkers.Add(1)
-	utils.PanicCapturingGo(func() {
-		defer svc.activeBackgroundWorkers.Done()
 
+	utils.ManagedGo(func() {
```

Comment:
I prefer this schema personally, but your summary of the changes in this PR says this fixes a panic and I'm not sure I see how. This feels like a 1:1 swap, am I missing something?